front/components/assistant_builder/ActionsScreen.tsx

```diff
@@ -113,16 +114,12 @@ export function hasActionError(
   }
 }
 
-type VaultIdToActions = Record<string, AssistantBuilderActionConfiguration[]>;
+type VaultIdToActions = Record<
+  string,
+  AssistantBuilderActionConfigurationWithId[]
+>;
 
-export default function ActionsScreen({
-  owner,
-  builderState,
-  setBuilderState,
-  setEdited,
-  setAction,
-  pendingAction,
-}: {
+interface ActionScreenProps {
```

Comment:
What's the rationale for using `interface` vs `type` here ?

Reply:
No strong rational except that it's a bit easier to read and less hacky, since types are very permissives.

Reply:
personally think it's easier to read when it consistent (so either using types everywhere or interfaces)